### PR TITLE
Clean-up `faq.rst`

### PR DESCRIPTION
**Pull request by Alnusjaponica**
## Motivation
Remove trailing spaces and fix a typo. 

## Description of the changes
Remove trailing spaces and fix a typo in `docs/faq.rst`

### CODE REVIEW

File: docs/source/faq.rst [hunk 1]
```diff
@@ -63,7 +63,7 @@ Below is an example that uses ``lambda``:
     study = optuna.create_study()
     study.optimize(lambda trial: objective(trial, min_x, max_x), n_trials=100)
 
-Please also refer to `sklearn_addtitional_args.py <https://github.com/optuna/optuna-examples/tree/main/sklearn/sklearn_additional_args.py>`_ example,
+Please also refer to `sklearn_additional_args.py.py <https://github.com/optuna/optuna-examples/tree/main/sklearn/sklearn_additional_args.py>`_ example,
```
Comment (not522) [inline docs/source/faq.rst:66]:
```suggestion
Please also refer to `sklearn_additional_args.py <https://github.com/optuna/optuna-examples/tree/main/sklearn/sklearn_additional_args.py>`_ example,
```

**Review by not522 — COMMENTED** [review]
Thank you for your PR. I'll merge after resolving my small comment.

**Comment by Alnusjaponica** [comment]
Thanks for the review. I fixed the typo.

**Review by not522 — APPROVED** [review]
LGTM

**Pull request merged**